index.js

```diff
@@ -311,7 +312,7 @@ module.exports = class HyperDHT extends DHT {
       }
     })
     await query.finished()
-    return { signature, seq }
+    return { query.closestNodes, signature, seq }
```

Comment:
```
closestNodes: query.closestNode
```

Reply:
This gives me:
`SyntaxError: Unexpected token '.'
[90m    at wrapSafe (internal/modules/cjs/loader.js:984:16)[39m
[90m    at Module._compile (internal/modules/cjs/loader.js:1032:27)[39m
[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)[39m
[90m    at Module.load (internal/modules/cjs/loader.js:933:32)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:774:14)[39m
[90m    at Module.require (internal/modules/cjs/loader.js:957:19)[39m
[90m    at require (internal/modules/cjs/helpers.js:88:18)[39m
    at Object.<anonymous> (D:\OODA_SPACE\oodaspacedev\hyperdhtoo\index2.js:1:13)
[90m    at Module._compile (internal/modules/cjs/loader.js:1068:30)[39m`

Reply:
logging `query` gives:
`mutablePut {
  query: <ref *1> Query {
    _events: [Object: null prototype] {},
    _eventsCount: 0,
    _maxListeners: undefined,
    _duplexState: 2171022,
    _readableState: ReadableState {
      stream: [Circular *1],
      queue: [FastFIFO],
      highWaterMark: 0,
      buffered: 0,...`

Reply:
I think we may need to access before `query.finished()`

Reply:
What exactly are you running? It should be:

```js
return { closestNodes: query.closestNodes, signature, seq }
```

Reply:
I'm unsure how to get data out of streamx objects...

Reply:
ok got it now! I should've read it properly!

Reply:
It's a query object (that inherits from stream). It's defined using that getter above. If you are getting a syntax error it means there is a typo somewhere.

Reply:
Was my bad, I was unfamiliar with getter in constructor syntax.